src/plugins/kibana_legacy/public/angular/angular_config.tsx

```diff
@@ -374,21 +383,21 @@ const $setupUrlOverflowHandling = (newPlatform: CoreStart, isLocalAngular: boole
     try {
```

Comment:
Doesn't have to happen in this PR, but we should probably pull the gist of these helpers out of the angular context and put them into generic helpers (e.g. for the ui capability redirect or the url overflow redirect)